src/components/NavigationBar/Navbar.module.scss

```diff
@@ -0,0 +1,175 @@
+@use "src/styles/colors" as colors;
```

Comment:
this is a high level request, unsure if it's doable, let me know if it's not something you think is doable

would it be possible to have comments about method for css as well? similar to how for code we generally try to comment the method by which we're doing something (a high level description), providing something like that would be really helpful for the css here as well, especially since this is providing a bit of sophistication w/ the animations and everything